service/src/java/org/apache/hive/service/server/HiveServer2.java

```diff
@@ -133,6 +133,8 @@
 import com.google.common.util.concurrent.SettableFuture;
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
 
+import javax.servlet.jsp.JspFactory;
```

Comment:
I see that you have removed javax.servlet dependency from root and service's pom.xml. I'm wondering where this dependency is coming from.

Reply:
Hi @saihemanth-cloudera, the jetty-runner jar contains the classes:
```
[Loaded javax.servlet.jsp.JspFactory from file:/opt/xxx/jetty-runner-9.4.48.v20220622.jar]
[Loaded javax.servlet.jsp.JspContext from file:/opt/xxx/jetty-runner-9.4.48.v20220622.jar]
[Loaded javax.servlet.jsp.PageContext from file:/opt/xxx/jetty-runner-9.4.48.v20220622.jar]
```
In fact this is where the class conflict happens.